Gemfile.lock

```diff
@@ -363,15 +366,15 @@ GEM
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 1.7)
-    rubocop-rails (2.5.0)
+    rubocop-rails (2.5.2)
       activesupport
       rack (>= 1.1)
       rubocop (>= 0.72.0)
     ruby-progressbar (1.10.1)
     ruby-vips (2.0.17)
       ffi (~> 1.9)
     safe_yaml (1.0.5)
-    sassc (2.2.1)
+    sassc (2.4.0)
```

Comment:
### [sassc](https://github.com/sass/sassc-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.1 → 2.4.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/sass/sassc-ruby/compare/v2.2.1...v2.4.0).

These are the first 10 commits:
- (5de84be) [Set appropriate encoding for source_map.](https://github.com/sass/sassc-ruby/commit/5de84bec0056f4e345259dd6c68c60d1ba990d3d)
- (7e2cdb5) [Bug fix for the commit 5de84be.](https://github.com/sass/sassc-ruby/commit/7e2cdb56b0df3d423fb3d7864c6f016082bf4cc0)
- (d492298) [Default --march-tune-native to false](https://github.com/sass/sassc-ruby/commit/d49229898c5c4e8a142129bdb5043bb01a479812)
- (681bcff) [always dispose data context](https://github.com/sass/sassc-ruby/commit/681bcffe894404d65596be3b03d0548fbe4e1223)
- (5c6c7ae) [allow passing functions directly](https://github.com/sass/sassc-ruby/commit/5c6c7aedec37a506144c1141663e2f200f2864fc)
- (08b80f8) [Update libsass to 3.6.2](https://github.com/sass/sassc-ruby/commit/08b80f83b14e5abcf32ba974968e1b3a1dee08e3)
- (eb02996) [disable lto flag for darwin + nix](https://github.com/sass/sassc-ruby/commit/eb02996ed1033816d6205f6aece5ada8fab71c1d)
- (dc0e251) [Update libsass to 3.6.3](https://github.com/sass/sassc-ruby/commit/dc0e2518b081c42b4e458c3e7fdf2635c24f5323)
- (30daccd) [Test on TruffleRuby in TravisCI](https://github.com/sass/sassc-ruby/commit/30daccd2dba6ffe55ab7a046c1e3461f42f05b44)
- (28aa5c7) [Use RbConfig::MAKEFILE_CONFIG['DLEXT'] instead of hardcoding…](https://github.com/sass/sassc-ruby/commit/28aa5c7102a253f61f34eceb74cf1564442865fb)


</details>